dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTableTest.java

```diff
@@ -96,10 +96,13 @@ private boolean areValuesEqual(final Identifier expectedResult, final Map<String
                 ((Date)result.get("create_date")).getTime()).toLocalDateTime().truncatedTo(
                 ChronoUnit.SECONDS);
 
-        return result.get("owner").equals(expectedResult.getOwner()) &&
+        return ((expectedResult.getOwner() == null && result.get("owner") == null) || result
```

Comment:
This looks very confusing. I'd try to refactor maybe creating small methods with good names.

Reply:
Done